Containers/nextcloud/start.sh

```diff
@@ -36,6 +36,34 @@ if ! [ -f "$NEXTCLOUD_DATA_DIR/this-is-a-test-file" ]; then
 fi
 sudo -u www-data rm -f "$NEXTCLOUD_DATA_DIR/this-is-a-test-file"
 
+# Install additional dependencies
+if [ -n "$ADDITIONAL_APKS" ]; then
+    if ! [ -f "/additional-apks-are-installed" ]; then
+        read -ra ADDITIONAL_APKS_ARRAY <<< "$ADDITIONAL_APKS"
+        for app in "${ADDITIONAL_APKS_ARRAY[@]}"; do
+            apk add "$app"
+        done
+    fi
+    touch /additional-apks-are-installed
+fi
+
+# Install additional php extensions
+if [ -n "$ADDITIONAL_PHP_EXTENSIONS" ]; then
+    if ! [ -f "/additional-php-extensions-are-installed" ]; then
+        read -ra ADDITIONAL_PHP_EXTENSIONS_ARRAY <<< "$ADDITIONAL_PHP_EXTENSIONS"
+        for app in "${ADDITIONAL_PHP_EXTENSIONS_ARRAY[@]}"; do
+            if [ "$app" = imagick ]; then
+                pecl install imagick-3.7.0
+                docker-php-ext-enable imagick
+            else
+                pecl install "$app"
+                docker-php-ext-enable "$app"
```

Comment:
It seems like that some extensions are not on pecl but instead delivered via core. If pecl fails, we need to run something like https://github.com/nextcloud/all-in-one/blob/b027cdd2933ede2f072fb4259147d32cc0276572/Containers/nextcloud/Dockerfile#L49 instead

Reply:
or maybe try to install it via core and afterwards via pecl. Need to figure out the best order.

Reply:
I guess best is to check if part of core by looking into `/usr/src/php/ext/`. If yes, install using docker-php-ext-install, if not, install using pecl.

Reply:
okay, just tested and it is not possible to check this beforehand based on a folder.

Reply:
found a solution